hs/src/LedgerState.hs

```diff
@@ -441,9 +441,19 @@ retirePools ls@(LedgerState _ ds _) epoch =
                & retiring .~ active)
   where (active, retiring') = Map.partition (epoch /=) (ds ^. retiring)
 
+-- |Calculate the change to the deposit pool for a given transaction.
+depositPoolChange :: LedgerState -> Tx -> Coin
+depositPoolChange ls tx = (currentPool + txDeposits) - txRefunds
```

Comment:
why the parentheses?

Reply:
it is possible that `txRefunds` > `txDeposits`, and so `txDeposits - txRefunds` is not a Natural. But, `currentPool + txDeposits >= txRefunds`, so `(currentPool + txDeposits) - txRefunds` is always fine.  I realize that leaving the parens off is the same as what I have written, but I thought it would be nice to write it this way to drive home the point. And hlint didn't complain. :) But I'm happy to remove them as well.

Reply:
I, I see. Is the evaluation order defined ? It doesn't make sense to risk an underflow, so please leave them and add a note